ui/rtk-query/telemetry.js

```diff
@@ -0,0 +1,58 @@
+import { api } from './index';
+
+const TAGS = {
+  GRAFANA: 'grafana',
+};
+
+const telemetryApi = api.injectEndpoints({
+  endpoints: (builder) => ({
+    getGrafanaBoards: builder.query({
+      query: ({ connectionID, grafanaBoardSearch }) => ({
+        url: `telemetry/metrics/grafana/boards/${connectionID}`,
+        params: { dashboardSearch: grafanaBoardSearch },
+        method: 'GET',
+        credentials: 'include',
+      }),
+      providesTags: () => [{ type: TAGS.GRAFANA }],
+    }),
+
+    getGrafanaConfig: builder.query({
+      query: () => ({
+        url: `telemetry/metrics/grafana/config`,
+        method: 'GET',
+        credentials: 'include',
+        headers: { 'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8' },
```

Comment:
we might not required to mention headers specifically. They are been set globally